entries/o/ocado.com.json

```
@@ -0,0 +1,15 @@
+{
+  "Ocado": {
+    "domain": "ocado.com",
+    "contact": {
+      "twitter": "Ocado",
+      "facebook": "ocado"
+    },
+    "keywords": [
+      "food"
```

Maybe add retail as well? :shrug: 
```suggestion
      "food",
      "retail"
```